common/cuchemcommon/fingerprint.py

```diff
@@ -10,16 +10,7 @@
 
 logger = logging.getLogger(__name__)
 
-try:
-    import cupy # TODO is there a better way to check for RAPIDS?
-except:
-    logger.info('RAPIDS installation not found. Numpy and pandas will be used instead.')
-    import numpy as xpy
-    import pandas as xdf
-else:
-    logger.info('RAPIDS installation found. Using cupy and cudf where possible.')
-    import cupy as xpy
-    import cudf as xdf
+import cupy
```

Comment:
Just out of curiosity, was this removed because it's no longer part of the benchmarking framework?

Reply:
yes, that is correct.

We can move benchmark modules out into its own git repo if we find it useful.

Reply:
Thanks for clarifying. Agreed, we may need to move into it's own repo. Suggest we wait until needed though, if that's OK.

Reply:
Please feel free to merge when ready.